packages/core/src/browser/navigatable.ts

```diff
@@ -87,7 +87,7 @@ export abstract class NavigatableWidgetOpenHandler<W extends NavigatableWidget>
     }
 
     protected serializeUri(uri: URI): string {
-        return uri.withoutFragment().toString();
+        return uri.withoutFragment().path.normalize().toString();
```

Comment:
it should return a serialized uri, not a path. Please use `withPath(path.normalize)` or add `normalizePath(): URI` to `URI` which does it

Reply:
@akosyakov Do you mean that finally, this line should be like
```return uri.withoutFragment().normalizePath().toString();```?

Reply:
And in `URI`, `normalizePath` would be like
```typescript
    normalizePath(): URI {
        return new URI(this.path.normalize().toString());
    }
```

Reply:
yes

Reply:
@akosyakov OK. Code updated now.